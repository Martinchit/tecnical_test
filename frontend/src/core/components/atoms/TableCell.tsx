import styled from 'styled-components';

export const TableCell = styled.td`
  vertical-align: middle !important;
  text-align: center;
`;
