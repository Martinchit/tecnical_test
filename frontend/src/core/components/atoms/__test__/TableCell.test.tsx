import React from 'react';
import { shallow } from 'enzyme';

import { TableCell } from '../TableCell';

describe('TableCell', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableCell />);
    expect(wrapper).toMatchSnapshot();
  });
});
