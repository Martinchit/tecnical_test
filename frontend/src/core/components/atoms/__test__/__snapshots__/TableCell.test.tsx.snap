// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableCell renders correctly 1`] = `
<td
  className="sc-AxjAm kLWhh"
/>
`;
